README.md

```diff
@@ -52,36 +52,30 @@ Say you have docker running on two hosts, accessible to each other as
 `$HOST1` and `$HOST2`, and want to deploy an application consisting of
 two containers, one on each host.
 
-On `$HOST1` run
-
-    host1$ weave launch
-    host1$ weave launch-dns
-    host1$ weave run --name a1 -ti ubuntu
-
-The first two lines start the weave router and DNS, each inside their
-own container - this needs to be done once per host. Whilst the
-required docker images are downloaded automatically, there is also a
-`weave setup` command for downloading all the images required for
-weave operation; this is a strictly optional step which is especially
-useful for automated installation of weave and ensures that any
-subsequent weave commands do not encounter delays due to image
-downloading.
-
-The third line runs our application container, allocating it an IP
-address on the weave network and registering it in DNS.  `weave run`
-invokes `docker run -d` behind the scenes, so we could be launching
-any container this way; here we just take a stock ubuntu container and
-launch a shell in it.  There's also a `weave start` command, which
-invokes `docker start` for starting existing containers.
-
-If our application consists of more than one container on this host we
-simply launch them with a variation on third second line.
+First start weave on $HOST1:
+
+    host1$ weave launch && weave launch-dns && weave launch-proxy
+
+this pulls and runs the router, DNS and proxy containers. Next we
```

Comment:
- we have lost the "this needs to be done once per host". Do you think that's obvious enough?
- "this runs the weave router, DNS and proxy, each in their own container".
- we do need to mention `weave setup` somewhere in our docs. It's quite an important command, used by many automated weave deployments.